src/resources/views/Email/passwordReset.blade.php

```diff
@@ -0,0 +1,14 @@
+@component('mail::message')
+# Récupération de mot de passe
+
+Vous avez fait une demande de récupération de mot de passe, pour réinitialiser votre mot de passe veuillez cliquer sur le bouton ci-dessous.
+
+@component('mail::button', ['url' => config('app.url') . '/reset-password?token=' . $token])
+Réinitialiser mdp
```

Comment:
Perhaps change `mdp` to something more explicit

Reply:
mdp is a common shortcup for "mot de passe" in French i don't think we need to change it as it's supposed to be a button so short text, also there's a description text above that explains to the user that is going to reset "son mot de passe"

Reply:
Whatever floats your boat